theano/tensor/blas.py

```diff
@@ -319,7 +322,9 @@ def default_blas_ldflags():
     # readily available. We try to see if that's the case, rather
     # than disable blas. To test it correctly, we must load a program.
     # Otherwise, there could be problem in the LD_LIBRARY_PATH.
-    return try_blas_flag(['-lblas'])
+    ret = try_blas_flag(['-lblas'])
+    if ret:
+        return ' '.join(ret)
```

Comment:
What I propose isn't equivalent. try_blas_flags modify what it return. We do not want that. In the past, the code was really returning "-lblas". I think we should continue to do that.